### ISSUE TITLE
Suggestion: Focus when clicking outside of the evaluation window

### ISSUE THREAD

**Issue by NitroG42** [Kyusung4698/PoE-Overlay#606]
Hello, first very good job with this program it works amazingly.

It has only one "feature" missing for me, and I'm not sure whether it can be fixed or not :
When you evaluate an object in-game, the evaluation popup is display and we can interact with it. If I want to make it diseappears I either press escape, or click outside of it.

With the Trade macro, clicking outside would directly click on the game, meaning for example if I long-clicked outside of the evaluation window, my character would start to move.

In PoE-Overlay, the first long click outside will do nothing. When I release the mouse button, if I click one more time my character will finally move.

**TLDR => we have to click twice outside of the evaluation window to make the game receive the input.**

Is there by any chance a way to make this happen (in a future release)?

**Comment by Kyusung4698** [comment]
## 0.6.21 (2020-04-12)

- add focus game after closing all dialogs (#606)
